app/src/main/res/values/colors.xml

```diff
@@ -26,5 +34,6 @@
 	<color name="grey_dark">#212121</color>
 	<color name="dim">#99000000</color>
 	<color name="shadow">#99000000</color>
+	<color name="transparent">#00FFFFFF</color>
```

Comment:
Useless. Use `@android:color/transparent` instead

Reply:
Oh, it's not even used anywhere, I forgot to remove it